tests/clients/test_dapr_grpc_client.py

```diff
@@ -681,6 +681,53 @@ def test_unlock_input_validation(self):
             with self.assertRaises(ValueError):
                 dapr.unlock(store_name, resource_id, invalid_input)
 
+    #
+    # Tests for workflow
+    #
+
+    def test_workflow(self):
+        dapr = DaprGrpcClient(f'localhost:{self.server_port}')
+        # Sane parameters
+        workflow_name = 'testWorkflow'
+        instance_id = str(uuid.uuid4())
+        workflow_component = "dapr"
```

Comment:
If the workflow component were something else (not `dapr`) this test would fail. This code here relies on the sidecar running - and that's not what we want for our unit tests. Only for integration tests!

I would like this to be a mocked test against the Mock gRPC server instead.

You need to stub out methods and provide expected responses here:
`tests/clients/fake_dapr_server.py`

The example in the examples folder can act as your integration test running against the real workflow engine. Those tests are always run with a real sidecar.